src/backend/db/products.js

```diff
@@ -8,23 +8,257 @@ import { v4 as uuid } from "uuid";
+  },
+  {
+    _id: uuid(),
+    productName:"Gucci-IntenseOud-UseMe-100ml (MEN)",
+    qunatity:"100 ml",
+    brand:"Gucci",
+    rating:"4.9",
+    IsRating:true,
+    originalPrice:"3000",
+    sellingPrice:"899",
+    discountOffer:"60% off",
+    IsSoldOut:true,
+    fastDelivery:false,
+    image:".assets/MEN/Gucci-IntenseOud-UseMe_men.png",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Gucci-oriental-base- 150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Gucci",
+    rating:"3.9",
+    IsRating:true,
+    originalPrice:"4000",
+    sellingPrice:"1399",
+    discountOffer:"40% off",
+    IsSoldOut:false,
+    fastDelivery:false,
+    image:".assets/MEN/gucci-oud_men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Zara-nuit-EAU-DE- 150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Zara",
+    rating:"2.7",
+    IsRating:false,
+    originalPrice:"2000",
+    sellingPrice:"1599",
+    discountOffer:"10% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/MEN/zara-nuit-men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Giorgio Armani Si Edp 150ml (WOMEN)",
+    qunatity:"150 ml",
+    brand:"Armani",
+    rating:"3.8",
+    IsRating:false,
+    originalPrice:"7600",
+    sellingPrice:"4000",
+    discountOffer:"30% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/WOMEN/Armani-Si_women.jpg",
+    categoryName: "Women",
+  },
+  {
+    _id: uuid(),
+    productName:"Giorgio Armani Si Edp 150ml (WOMEN)",
+    qunatity:"150 ml",
+    brand:"Armani",
+    rating:"4",
+    IsRating:true,
+    originalPrice:"7000",
+    sellingPrice:"2599",
+    discountOffer:"20% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/WOMEN/ArmaniSIFioriforWomen.jpg",
+    categoryName: "Women",
+  },
+  {
+    _id: uuid(),
+    productName:"Modern-muse-estee-lauder- 150ml (WOMEN)",
+    qunatity:"150 ml",
+    brand:"Estee Lauder",
+    rating:"3.1",
+    IsRating:false,
+    originalPrice:"1000",
+    sellingPrice:"799",
+    discountOffer:"40% off",
+    IsSoldOut:false,
+    fastDelivery:false,
+    image:".assets/WOMEN/MODERN-MUSE-ESTEE-LAUDER-women.jpg",
+    categoryName: "Women",
+  },
+  {
+    _id: uuid(),
+    productName:"Estee-Lauder-Pleasures-100ml (WOMEN)",
+    qunatity:"100 ml",
+    brand:"Estee Lauder",
+    rating:"4",
+    IsRating:true,
+    originalPrice:"5000",
+    sellingPrice:"1599",
+    discountOffer:"40% off",
+    IsSoldOut:true,
+    fastDelivery:true,
+    image:".assets/WOMEN/Estee-Lauder-Pleasures-For-Women_.jpg",
+    categoryName: "Women",
+  },
+  {
+    _id: uuid(),
+    productName:"Zara-women-fruity-100ml (WOMEN)",
+    qunatity:"100 ml",
+    brand:"Zara",
+    rating:"4.2",
+    IsRating:true,
+    originalPrice:"5000",
+    sellingPrice:"999",
+    discountOffer:"70% off",
+    IsSoldOut:true,
+    fastDelivery:true,
+    image:".assets/WOMEN/zara-fruity-women.jpg",
```

Comment:
it's a just a suggestion, it would be better if you provide ./assets in all image